docs/LogGuide.md

```diff
@@ -952,69 +1546,120 @@ such as coil turns where there is a zone seal.
+[14:11:35.0] 22:34:2021-04-26T14:11:35.0000000-04:00:[id]:[name]:[targetId]:[targetName]:[toggle]
+```
+
+#### Examples
 
 ```log
-22:105E3321:Tini Poutini:105E3321:Tini Poutini:01
-22:40018065:Twintania:40018065:Twintania:00
+Network Log Line Examples:
+34|2021-04-26T14:19:48.0400000-04:00|4001C51C|Dragon's Head|4001C51C|Dragon's Head|00|a7248aab1da528bf94faf2f4b1728fc3
+34|2021-04-26T14:22:19.1960000-04:00|4000B283|Selene|4000B283|Selene|01|734eef0f5b1b10810af8f7257d738c67
+34|2021-04-26T14:25:06.0720000-04:00|4000B262|Midgardsormr|4000B262|Midgardsormr|00|e3482e01bb9e1c358c23a843b6c63556
+34|2021-04-26T14:18:00.9980000-04:00|10FF0001|Tini Poutini|10FF0001|Tini Poutini|00|5b840ff5e79d908b3f9a3e806a64fd0b
+
+ACT Log Line Examples:
+[14:19:48.40] 22:34:2021-04-26T14:19:48.0400000-04:00:4001C51C:Dragon's Head:4001C51C:Dragon's Head:00:a7248aab1da528bf94faf2f4b1728fc3
+[14:22:19.196] 22:34:2021-04-26T14:22:19.1960000-04:00:4000B283:Selene:4000B283:Selene:01:734eef0f5b1b10810af8f7257d738c67
+[14:25:06.72] 22:34:2021-04-26T14:25:06.0720000-04:00:4000B262:Midgardsormr:4000B262:Midgardsormr:00:e3482e01bb9e1c358c23a843b6c63556
+[14:18:00.998] 22:34:2021-04-26T14:18:00.9980000-04:00:10FF0001:Tini Poutini:10FF0001:Tini Poutini:00:5b840ff5e79d908b3f9a3e806a64fd0b
 ```
 
+<!-- AUTO-GENERATED-CONTENT:END -->
+
 ### 23: NetworkTether
 
 This log line is for tethers between enemies or enemies and players.
 This does not appear to be used for player to player skill tethers like dragonsight or cover.
 (It can be used for enemy-inflicted player to player tethers such as burning chains in Shinryu N/EX.)
 
-Structure:
-`23:[SourceId]:[SourceName]:[TargetId]:[TargetName]:[Unknown1 (4 bytes)]:[Unknown2 (4 bytes)]:[Type (4 bytes)]:[TargetId]:[Unknown3 (4 bytes)]:[Unknown4 (4 bytes)]:`
+<!-- AUTO-GENERATED-CONTENT:START (logLines:type=Tether&lang=en) -->
 
-Examples:
+#### Structure
+
+```log
+Network Log Line Structure:
+35|2021-04-26T14:11:35.0000000-04:00|[sourceId]|[source]|[targetId]|[target]|||[id]
+
+ACT Log Line Structure:
+[14:11:35.0] 23:35:2021-04-26T14:11:35.0000000-04:00:[sourceId]:[source]:[targetId]:[target]:::[id]
+```
+
+#### Regexes
 
 ```log
-23:40015B4E:Weapons Node:40015B4D:Gravity Node:751E:0000:000E:40015B4D:000F:7F4B:
-23:4000E84B:Zu Cockerel:1048638C:Tini Poutini:0000:0000:0006:1048638C:000F:7FEF:
-23:40001614:Omega:10532971:Potato Chippy:0023:0000:0054:10532971:000F:0000:
+Network Log Line Regex:
+^(?<type>(?:33))\|(?<timestamp>(?:[^|]*))\|(?<instance>(?:[^|]*))\|(?<command>(?:[^|]*))\|(?<data0>(?:[^|]*))\|(?<data1>(?:[^|]*))\|(?<data2>(?:[^|]*))\|(?<data3>(?:[^|]*))\|
+
+ACT Log Line Regex:
+(?<timestamp>(?:^.{14})) 21:(?<instance>(?:.*?)):(?<command>(?:.*?)):(?<data0>(?:.*?)):(?<data1>(?:.*?)):(?<data2>(?:.*?)):(?<data3>(?:.*?))$
 ```
 
+#### Examples
+
+```log
+Network Log Line Examples:
+35|2021-04-26T17:27:07.0310000-04:00|40003202|Articulated Bit|10FF0001|Tini Poutini|0000|0000|0001|10029769|000F|0000||ad71d456437e6792f68b19dbef9507d5
```

Comment:
Probably only 2-3 examples would be enough here.

Reply:
One of the examples here was specifically to demonstrate that the names can end up blank. Maybe we should have a note somewhere else in the document stating that the source/target name can be blank on many lines, and they're only guaranteed to be present on, e.g., the 03 line?

Reply:
That might be a good just to note up at the top in general.